:recycle: Add mse home (#19)
Cosmian/mse-cli: README.md

- `MSE_PCCS_URL` to specify another PCCS URL 

## Usage - MSE Home

```console
$ mse home -h
```

Note: if you set the env variable `MSE_BACKTRACE=full`, a Python stacktrace will be printed in case of errors.

You can find below the use flow step by step.

### Scaffold your app

__User__: the code provider

```console
$ mse home scaffold example
```

### Test your app, your docker and your msehome configuration

__User__: the code provider

```console
$ mse home localtest --project example/
```

### Create the MSE package with the code and the docker image

__User__: the code provider

```console
$ mse home package --project example/ \
                   --output workspace/code_provider 
```

The generated package can now be sent to the sgx operator.

### Spawn the MSE docker

__User__: the SGX operator

```console
$ mse home spawn --host myapp.fr \
                 --port 7777 \
                 --size 4096 \
                 --package workspace/code_provider/package_mse_src_1683276327723953661.tar \
                 --output workspace/sgx_operator/ \
                 app_name
```

> Now, evidences have been automatically collected and the microservice is up.

Reviewer: ```suggestion
At this moment, evidences have been automatically collected and the microservice is up.
```

```console
$ mse home verify --package workspace/code_provider/package_mse_src_1683276327723953661.tar \
                  --evidence output/evidence.json \
                  --output /tmp
```

If the verification succeeds, you get the RA-TLS certificate (written as a file named `ratls.pem`) and you can now seal the code key to share it with the SGX operator.

### Seal your secrets

__User__: the code provider

```console
$ mse home seal --secrets example/secrets_to_seal.json \
                --cert /tmp/ratls.pem \
                --output workspace/code_provider/
```

### Finalize the configuration and run the application

__User__: the SGX operator

```console
$ mse home run --sealed-secrets workspace/code_provider/secrets_to_seal.json.sealed \
               app_name
```

### Test the deployed application

__User__: the SGX operator

```console
$ mse home test --test workspace/sgx_operator/tests/ \
                --config workspace/sgx_operator/mse.toml \
                app_name
```

### Decrypt the result

__User__: the code provider

Assume the SGX operator gets a result as follow: `curl https://localhost:7788/result --cacert /tmp/ratls.pem > result.enc`

> Then, the code provider can decrypt the result has follow:

Reviewer: ```suggestion
Then, the code provider can decrypt the result as follow:
```

```console
$ mse home decrypt --key key.txt \
                   --output workspace/code_provider/result.plain \
                   result.enc
$ cat workspace/code_provider/result.plain
```

> ### Manage the mse docker

Reviewer: ```suggestion
### Manage the MSE docker
```

__User__: the SGX operator

You can stop and remove the docker as follow:

```console
$ mse home stop [--remove] <app_name>
```

You can restart a stopped and not removed docker as follow:

```console
$ mse home restart <app_name>
```

> You can get the mse docker logs as follow:

Reviewer: ```suggestion
You can get the MSE docker logs as follow:
```

```console
$ mse home logs <app_name>
```

> You can get the mse docker status as follow:

Reviewer: ```suggestion
You can get the MSE docker status as follow:
```

```console
$ mse home status <app_name>
```

> You can get the list of running mse dockers:

Reviewer: ```suggestion
You can get the list of running MSE dockers:
```
